mains/Scrambled eggs with tomatoes.md

```diff
@@ -0,0 +1,39 @@
+# Scrambled eggs with tomatoes recipe
+
+
+## Ingredients
+
+- **1-2** eggs per person
+- **1-2** tomatoes per person
+- **1 tablespoon** Salt
+- **1-2 tablespoon** Soy sauce
+- **2 tablespoon** Soy sauce is you like more.(optional)
+- **1** Small green onion
+- **1-2 tablespoon** Sugar or more salt based on your taste.(Optional)
+- **1 tablespoon** oyster sauce.
+- **1 spoon** Oil.
+- **
+## Instructions
+1. **Prepare eggs and tomatoes:**
+   - Beat the eggs until thoroughly mixed.
+   - Add a tablespoon salt to eggs and mix well.
+   - Add a tablespoon soy sauce to eggs and mix well if you like (optional).
+   - Peel the tomatoes if you like and chop tomatoes into cubes.
+   - Chop the green onions into small sections.
+2. **Preheat wok:**
+   - Preheat your wok over high heat for 10 sec.
+3. **Add Oil:**
+   -  Add oil and heat for 5 sec.
+4. **Fry eggs:**
+   - Add eggs and fry until 80% cooked.
+   - Take eggs in a plate.
+5. **Fry tomatoes:**
+   - Add tomato cubes and fry for 10 sec.
+5. **(This is an error need to be fixed)?!$?????**

```

Comment:
```suggestion
```
There seems to be double step 5, only one of which looks necessary. I think this can be removed